### PR TITLE
Horizontal bar chart with axis Component and Unit Tests

### PR DESCRIPTION
**Pull request by v-baambati**
1.Added unit test cases.
Code coverage before UTs
HorizontalBarChartWithAxis_before
Code Coverage After UTs
HorizontalBarChartWithAxis_AfterUTS

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f90a7d66c47ffca2e307fb396d87540e0c1f7a1e'>f90a7d66c47ffca2e307fb396d87540e0c1f7a1e</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxisRTL.test.tsx [hunk 1]
```diff
@@ -1,14 +1,472 @@
...
+import { DefaultPalette } from '@fluentui/react/lib/Styling';
+import { HorizontalBarChartWithAxis } from './HorizontalBarChartWithAxis';
+import { getByClass, getById, testWithWait, testWithoutWait } from '../../utilities/TestUtility.test';
+import { HorizontalBarChartWithAxisBase } from './HorizontalBarChartWithAxis.base';
+import { IHorizontalBarChartWithAxisDataPoint } from '../../HorizontalBarChart';
 import { toHaveNoViolations } from 'jest-axe';
 
 expect.extend(toHaveNoViolations);
 
+const chartPoints: IHorizontalBarChartWithAxisDataPoint[] = [
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxisRTL.test.tsx:16]: move the data to the common utilities/test-data.ts file

**Comment by AtishayMsft** [comment]
There seems to be duplicate changes in this PR with https://github.com/microsoft/fluentui/pull/29064 
Can you pull latest master and update this PR appropriately.

**Comment by v-baambati** [comment]
https://github.com/microsoft/fluentui/pull/29064 completed but that code is not available in Matser branch so i have added same files in this branch

**Comment by AtishayMsft** [comment]
You need to add a changelog file for the change in test-data.ts file.

**Comment by v-baambati** [comment]
@AtishayMsft added change log file for this PR.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**